spec.md

```diff
@@ -0,0 +1,238 @@
+
+Artifact layer types follow the same convention as manifests, utilizing the `layer.mediaType` with the following format:  
+`application/vnd.`[org|company]`.`[layerType]`.`[layerSubType]`.layer.`[version]`+`[fileFormat]
+
+### Example Layer Types
+
+|Artifact Type|mediaType|
+|-|-|
+|OCI Image|`application/vnd.oci.image.layer.v1+tar`|
+|Docker Image|`application/vnd.docker.image.rootfs.diff.tar.gzip`|
+|Helm Chart|`application/vnd.cncf.helm.chart.layer.v1+tar`|
+|Singularity SIF Layer|`application/vnd.sylabs.sif.layer.v1+tar`|
+
+### One or More Layer Types
+
+> NOTE: Should this section be in the `spec.md`, or in an `artifact-authors.md` doc?
+
+The number of layer types is up to the artifact author. Some things to consider when designing the layer format:
+
+- **Continuos Builds:** will developers automate building and pushing the artifact to a registry? For each rebuild, what will change? Consider the docker layering model, where users benefit from base layers that don't change often. If only a small piece of content changes, can you separate that content into a unique layer?
+- **Layer reuse**: when layers are pushed to a registry, layers can be de-duped. If your artifact type may benefit from shared layers across many artifact instances, consider splitting up the layers to those that change often, and those that don't.
+- **Layer sizes**: does your artifact have large content that may benefit from being split up in to smaller elements that can be concurrently uploaded and/or downloaded to a registry?
+- **Content and Config**: While a general best practice for container images says the image in the registry should not contain environmental configuration, you may be choosing to push environmental configuration to a registry as it's own artifact. If the same artifact is pushed to the same registry multiple times, only unique by some configuration, consider splitting up the configuration into its own layer, to benefit from layer reuse.
+
+### Shared Layer Example
+
+Consider an Azure Resource Manager (ARM) or AWS Cloud Formation template stored in a registry. You may have the base template stored for shared team use. As teams deploy the template, consider storing the parameters file as a separate blobs. When pushed to a registry, most registry operators will de-dupe the base template, only storing the unique parameters blob for each artifact.
+
+## Optional: Defining Config Schema
+
+While the value of `manifest.config.mediaType` is used to determine the artifact type, the persistance of a `config.json` file is OPTIONAL. Artifacts can push a null reference for `config.json` persistance.
+
+When defining an artifact type, the persistance of the artifact may be broken up into content and configuration. Configuration can be stored as a blob, or it can be stored and referenced by the `manifest.config`.
```

Comment:
`manifest.config` is a descriptor pointing at a blob.  What distinction are you trying to make?

Reply:
Please see above if that helps clarify what we’re doing wit the config object.